### PR TITLE
i18n cleanup, round 2

### PR DESCRIPTION
**Pull request by eyeseast**
Cleans up all the translations we're no longer using, plus old language files.

- [x] English
- [x] Spanish
- [x] French
- [x] Italian
- [x] German
- [x] Russian
- [x] Ukrainian

### CODE REVIEW

File: src/langs/json/it.json [hunk 1]
```diff
@@ -1,45 +1,41 @@
...
+    "explore": "Esplora",
+    "edit": "Cambia",
     "search": "Cerca",
-    "feedback": "Feedback"
+    "feedback": "Feedback",
+    "title": "Titolo"
   },
   "search": {
-    "matchingResults": "{n, plural, one {# risultato corrispondente} other {# risultati correlati}}"
+    "matchingResults": "{n, plural, one {# matching result} other {# matching results}}"
```
Comment (allanlasser) [inline src/langs/json/it.json:14]: This change looks wrong to me, since this will now render the text in English.

**Review by allanlasser — CHANGES_REQUESTED** [review]

File: src/langs/json/ru.json [hunk 2]
```diff
@@ -1,706 +1,568 @@
...
-    "title": "Заголовок",
-    "actions": "Действия",
-    "debug": "Отладка",
-    "viewErrors": "Просмотр ошибок",
-    "viewS3": "Просмотр в S3"
+    "title": "Повторить реанимирование",
+    "reprocessDocs": "Проceeding forcedly will force the following {n, plural, one {выбранный документ} other {представлено {selected} dokumentа}} to reprocess page and image text:",
+    "reprocessSingleDoc": "Проceedяте, это приведет к рендерингу документа, который forcing reprocess page и image тексту",
+    "continue": "Но хотите продолжить?",
+    "confirm": "Репrocess"
```
Comment (allanlasser) [inline src/langs/json/ru.json:73]: These are kinda mish-mash of English and Russian

File: src/langs/json/ru.json [hunk 3]
```diff
@@ -1,706 +1,568 @@
...
-    "fullscreenOption": "Функция полноэкранного режима",
-    "fsVisibleHelp": "Показывать значок полноэкранного режима в правом нижнем углу",
-    "fsHiddenHelp": "Скрыть опцию полноэкранного режима",
+    "responsive": "Ответственный",
+    "respOn": "ON",
+    "respOnHelp": "Наполните ширину контейнера статьи (обычно funciona лучше)",
+    "respOff": "Оф",
+    "respOffHelp": "Документ будет соответствовать размерам указанным ниже",
+    "width": "Širina",
+    "widthAuto": "Понимает как filled the available space, if the document is set to responsive, otherwise defaults to 700px.",
```
Comment (allanlasser) [inline src/langs/json/ru.json:82]: mish-mash

File: src/langs/json/ru.json [hunk 4]
```diff
@@ -1,706 +1,568 @@
...
+    "sbVisibleHelp": "Показывать сайдбар по замовчанню",
+    "titleBehavior": "Навыки 제목а",
+    "tVisibleHelp": "Показать عنوان и авторство в шапке просмотра",
+    "tHiddenHelp": "Скрыть título в заголовке просмотра (еще видимый в Sidebar)",
+    "pdfLink": "Ссылка на PDF",
+    "plVisibleHelp": "Показать ссылку на исходный PDF-документ в правом крае",
+    "plHiddenHelp": "Скрыть ссылку на PDF из сайдбара (файл по-прежнему доступен, если URL известна)",
+    "fullscreenOption": "Полноэкранная опция",
+    "fsVisibleHelp": "Показывать(fullscreen) иконку в правом нижнем углу",
+    "fsHiddenHelp": "Закрыть опцию(fullscreen)",
```
Comment (allanlasser) [inline src/langs/json/ru.json:102]: Losing faith in the Russian LLM output…

**Review by allanlasser — CHANGES_REQUESTED** [review]
Caught some issues with interpolated strings, tried to correct them. The Russian is wacky, with Arabic and Chinese characters sneaking in, which caused me to lose faith in this approach for non-Latin languages. But this is decent as a first pass.

I'd like to see a second attempt that uses the Google Translate API and does a finer parsing of the strings to extract nested strings…but that can happen in a follow-up.

The last thing we need is a README update explaining how to update translations.

File: src/langs/json/ru.json (hunk 4 above)
Comment (eyeseast) [reply src/langs/json/ru.json:102]: Yeah, Russian is hard. Might actually see if there's a better model for non-Latin languages.

**Comment by eyeseast** [comment]
I reverted and cleaned up the Russian translation, and added a small readme.

**Review by allanlasser — APPROVED** [review]

**Pull request merged**